src/native/corehost/hostpolicy/coreclr.cpp

```diff
@@ -20,6 +20,13 @@ namespace
     }
 }
 
+void coreclr_t::log_error(const char* line, void* arg)
```

Comment:
put this into an anonymous namespace - it doesn't need to be part of the class - and thus can be removed from the header.

Reply:
Done